components/installer/README.md

```diff
@@ -15,96 +14,94 @@ The project is based on the Kubernetes operator pattern. It tracks changes of th
 ## Development
 
 Before each commit, use the [before-commit.sh](./before-commit.sh) script to test your changes:
-```
-./before-commit.sh
-```
+  ```
+  ./before-commit.sh
+  ```
 
-### Build a Docker image
+## Build a Docker image
 
-Run the [build.sh](./scripts/build.sh) script to build a Docker image:
+Run the [build.sh](./scripts/build.sh) script to build a Docker image of the Installer:
+  ```
+  ./scripts/build.sh
+  ```
 
-```
-./scripts/build.sh
-```
+## Run on Minikube using local sources
 
-### Run on Minikube
+Export the path to Kyma sources as an environment variable. This environment variable is used to trigger shell scripts located in Kyma.
+  ```
+  export KYMA_PATH={PATH_TO_KYMA_SOURCES}
+  ```
 
-#### Run with local Kyma resources
-```
-export KYMA_PATH={PATH_TO_KYMA_SOURCES}
-```
-This environment variable is used to trigger shell scripts located in Kyma.
-```
-./scripts/run.sh --local --cr {PATH_TO_CR}
-```
-
-It will run Minikube, set up the installer, and install Kyma from local sources. If Minikube was started before, it will be restarted.
+Run this script to run Minikube, set up the installer, and install Kyma from local sources. If Minikube was started before, it will be restarted.
```

Comment:
```suggestion
Use this script to run Minikube, set up the Installer, and install Kyma from local sources. If Minikube was started before, it will be restarted.
```